doc_src/terminal-compatibility.rst

```diff
@@ -165,13 +165,17 @@ Optional Commands
      -
      - Select background color from 24-bit RGB colors.
      -
+   * - ``\e[49m``
+     -
+     - Reset background color to the terminal's default.
+     -
```

Comment:
Not sure if this is in terminfo. I guess I'll test this on some 40 terminals:

    set_color red --background blue; set_color --background=normal; echo red-on-default